src/block-management/block-holder.js

```diff
@@ -162,12 +150,13 @@ export class BlockHolder extends React.Component<PropsType, StateType> {
 			return null;
 		}
 
+		const order = this.props.getBlockIndex( this.props.clientId, this.props.rootClientId );
 		return (
 			<InlineToolbar
-				clientId={ this.props.clientId }
 				onButtonPressed={ this.onInlineToolbarButtonPressed }
 				canMoveUp={ ! this.props.isFirstBlock }
 				canMoveDown={ ! this.props.isLastBlock }
+				rowIndex={ order }
```

Comment:
This is not necessary, we can connect `InlineToolbar` to the block-editor store to retrieve the block order:
```js
withSelect( ( select, { clientId } ) => {
	const {
		getBlockOrder,
	} = select( 'core/block-editor' );

	return {
		order: getBlockOrder( clientId ),
	};
} )( InlineToolbar );
```

Reply:
I changed this to the clientId and updated the InlineToolbar, this what you meant? I used `getBlockIndex` instead since `getBlockOrder` wasn't returning what I expected?

Reply:
Oh right, getBlockOrder returns a list of clientIds, my bad, looks good now :)